quickstart/examples/llm-d-sim/README.md

```diff
@@ -0,0 +1,136 @@
+# Quickstart - LLM-D Simulation Deployment
+
+This is a simulation example that demonstrates how to deploy using the llm-d-infra system with the `ghcr.io/llm-d/llm-d-inference-sim` image. This example simulates inference responses and can run on minimal resources without requiring actual GPU hardware.
+
+## Installation
+
+> To adjust the simulation settings or any other modelservice values, simply change the values.yaml file in [ms-llm-d-sim/values.yaml](ms-llm-d-sim/values.yaml)
+
+1. Install the dependencies; see [install-deps.sh](../../install-deps.sh)
+2. Use the quickstart to deploy Gateway CRDS + Gateway provider + Infra chart:
+
+```bash
+# From the repo root
+cd quickstart
+HF_TOKEN=$(HFTOKEN) ./llmd-infra-installer.sh --namespace llm-d -r infra-llm-d-sim --gateway kgateway
+```
+
+    - It should be noted release name `infra-llm-d-sim` is important here, because it matches up with pre-built values files used in this example.
+
+3. Use the helmfile to apply the modelservice and GIE charts on top of it.
+
+```bash
+cd examples/llm-d-sim
+helmfile --selector managedBy=helmfile apply helmfile.yaml
+```
+
+## Verify the Installation
+
+1. Firstly, you should be able to list all helm releases to view all 5 charts that should be installed:
+
+```bash
+helm list --all-namespaces --all --debug
+NAME         	NAMESPACE      	REVISION	UPDATED                                	STATUS  	CHART                    	APP VERSION
+gaie-simple  	llm-d          	1       	2025-07-17 04:20:52.060094662 +0000 UTC	deployed	inferencepool-v0.4.0     	v0.4.0
+infra-simple 	llm-d          	1       	2025-07-17 03:14:42.090423968 +0000 UTC	deployed	llm-d-infra-1.0.5        	0.1
+kgateway     	kgateway-system	4       	2025-07-17 03:14:34.193133129 +0000 UTC	deployed	kgateway-v2.0.3          	1.16.0
+kgateway-crds	kgateway-system	4       	2025-07-17 03:14:32.878851899 +0000 UTC	deployed	kgateway-crds-v2.0.3     	1.16.0
+ms-simple    	llm-d          	1       	2025-07-17 04:20:52.086321656 +0000 UTC	deployed	llm-d-modelservice-0.0.10	0.0.1
+```
+
+Note: if you chose to use `istio` as your Gateway provider you would see those (`istiod` and `istio-base` in the `istio-system` namespace) instead of the kgateway based ones.
+
+2. Find the gateway service:
+
+```bash
+kubectl get services
+NAME                             TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)             AGE
+gaie-simple-epp                  ClusterIP   10.101.43.58    <none>        9002/TCP,9090/TCP   29m
+infra-simple-inference-gateway   NodePort    10.104.22.184   <none>        80:31233/TCP        95m
```

Comment:
Again these services will be derived from the release name above, so I would assume none of these docs are updated for this example. Its no big deal we can fix this later but something we should keep in mind

Reply:
Changed to `sim`. Ty!